### PR TITLE
add self_delimiting? field

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

File: lib/membrane_opus.ex [hunk 1]
```diff
@@ -20,12 +20,9 @@ defmodule Membrane.Opus do
 
   @type t :: %__MODULE__{
           bitrate: bitrate_t,
-          channels: channels_t
+          channels: channels_t,
+          self_delimiting?: boolean
```
Comment (mickel8) [inline lib/membrane_opus.ex]: We have documented what are `bitrate` and `channels` so maybe let's do sth similar with `self_delimiting?`.

File: lib/membrane_opus.ex [hunk 2]
```diff
@@ -20,12 +20,9 @@ defmodule Membrane.Opus do
...
         }
 
-  @enforce_keys [
-    :bitrate,
-    :channels
-  ]
-  defstruct @enforce_keys
+  defstruct [:bitrate, :channels, self_delimiting?: false]
```
Comment (mickel8) [inline lib/membrane_opus.ex]: Shouldn't there be any defaults?

**Review by mickel8 — CHANGES_REQUESTED** [review]
We - missclick

File: lib/membrane_opus.ex (hunk 2 above)
Comment (mat-hek) [reply lib/membrane_opus.ex]: The defaults are `nil`, meaning they are unknown

**Review by mickel8 — APPROVED** [review]

**Pull request merged**